backend/ormconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "mysql",
+  "host": "localhost",
+  "port": 3306,
+  "username": "root",
+  "password": "your_password",
+  "database": "your_database",
+  "entities": ["entities/*.ts"],
```

Comment:
entities can have .entity.ts type instead of saying it will only be present in entities folder?

Reply:
done